server/src/test/scala/za/co/absa/atum/server/api/TestData.scala

```diff
@@ -35,6 +35,16 @@ trait TestData {
   protected val partitioningSubmitDTO3: PartitioningSubmitDTO =
     partitioningSubmitDTO1.copy(authorIfNew = "yetAnotherAuthor")
 
+  val additionalDataDTO: AdditionalDataDTO = Map(
+    "key1" -> Some("value1"),
+    "key2" -> None,
+    "key3" -> Some("value3")
+  )
+  // Partitioning DTO
+  protected val partitioningDTO1: PartitioningDTO = Seq.empty
+  protected val partitioningDTO2: PartitioningDTO = Seq.empty
```

Comment:
Having all the same is kind of pointless and may cause problems in mocks
```suggestion
  protected val partitioningDTO1: PartitioningDTO = Seq(PartitionDTO("key1", "value1"))
  protected val partitioningDTO2: PartitioningDTO = Seq(PartitionDTO("key2", "value2"), PartitionDTO("key3", "value3"))
```